index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body class="all">
+    <header>
+        <nav>
+            <img class="navbar" src="./images/Bars.svg" alt="bars"> 
+        </nav>
+    </header>
+    <section class="section-spacing">
+        <div class="introduction">
+            <div class="texts">
+                <h1 class="welcome">Hey there.<br/> I'm Assem.<br/>
+                  <span class="green">I'm a software developer</span></h1>
+                <p class="profession"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                  don't hesitate to contact me.</p>
+            </div>
+            <div class="socialmedia">
+                <ul>
+                    <li><a href="#"><img src="./images/twitter.svg" alt="Twitter"></a></li>
+                    <li><a href="#"><img src="./images/Linkedin icon.svg" alt="Linkedin"></a></li>
+                    <li><a href="#"><img src="./images/medium.svg" alt="Medium"></a></li>
+                    <li><a href="#"><img src="./images/github.svg" alt="Github"></a></li>
+                    <li><a href="#"><img src="./images/angellist.svg" alt="Enabled"></a></li>
+                </ul>
+            </div>
+            <div class="scroll">
+                <img src="./images/Disabled.png" alt="scroll button">
+            </div>
+        </div>
+    </section>
+    <section class="works">
+        <div class="recent-works">
+           <h2>My Recent Works</h2>
+           <img class="indicator" src="./images/Indicator.svg" alt="Indicator">
+        </div>
+        <div class="individual-card-item-flex">
+            <div class="card-work-item">
+                <div class="individual-card-item">
+                    <h3>Multi-Post Stories</br>Gain+Glory</h3>
+                    <ul class="languages">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="btn">See project</button>
```

Comment:
Great job on implementing the button functionality! However, there are some improvements needed to match the design specifications for the button styles.

- [ ] Currently, the default, active, and hover styles of the button do not closely resemble the design. It's important to align the button styles with the provided design to maintain consistency and visual coherence.
![image](https://github.com/zhorabay/My-portfolio/assets/55840999/1e691e3b-70b5-4f70-ad40-d5c0d52b124d)
I recommend carefully reviewing the design and updating your code to ensure that the button styles, including color, background, font styles, and any additional effects, accurately match the design specifications. By closely following the design, you will create a more polished and professional look for your button. Keep up the good work! 👍

**_Note: This request applies for all buttons in the project. 👍_**